airflow/contrib/hooks/gcp_kms_hook.py

```diff
@@ -39,6 +42,9 @@ class GoogleCloudKMSHook(GoogleCloudBaseHook):
     """
     Interact with Google Cloud KMS. This hook uses the Google Cloud Platform
     connection.
+
```

Comment:
It seems to me that this is not true, because no method in this class needs project_id

Reply:
Thanks for that!